client/galaxy/scripts/components/RuleCollectionBuilder.vue

```diff
@@ -1987,19 +1987,19 @@ export default {
 <style>
 .table-column {
     width: 100%;
-    /* overflow: scroll; */
+    overflow: hidden;
```

Comment:
Now that I look at this a second time -- are we sure that this fairly generic classname doesn't leak and affect anything else?  This block isn't scoped.